picard/album.py

```diff
@@ -534,8 +531,12 @@ def column(self, column):
                 unsaved = self.get_num_unsaved_files()
                 if unsaved:
                     text += '; %d*' % (unsaved,)
-                text += ungettext("; %i image", "; %i images",
-                                  len(self.metadata.images)) % len(self.metadata.images)
+                if getattr(self.metadata, 'has_common_images', True):
+                    text += ungettext("; %i image", "; %i images",
+                                      len(self.metadata.images)) % len(self.metadata.images)
```

Comment:
`len(self.metadata.images)` is repeated 4 times, please store the value in a short variable.